## Title: The adaptive potential of Southern Ocean phytoplankton
#### Principal Investigator: Ian Bishop

# Panel summary
###### *Overview*

This proposal focused on the phenotypic diversity of diatoms, and how epigenetics may impact their adaptive potential. This type of analysis has never been done on eukaryotic photoautotrophs. They will use ddRAD-seq to analyze diversity, and reduced representation bisulfite sequencing to examine epigenetic variability. They hope that elucidating these pathways will shed light on the possibility of diatoms adapting to future climate change.

###### *Intellectual merit*

*Strengths*

A major strength of this proposal was that it fills a clear knowledge gap. Panelists agreed that looking at epigenetic dynamics in these populations is a novel and interesting idea. There is also a clear connection to climate change research, which has important implications for future conservation efforts. This project will also examine two different species, which will allow the researchers to examine ecological and evolutionary dynamics and make stronger conclusions. While no reference genome exists for the species in question, the researcher has clearly examined the problem and found a good solution through various RAD-seq methods.

*Weaknesses*

The main weakness the panelists identified was a lack of some key background information. First of all, it would be helpful to acknowledge the debate on whether or not phenotypic plasticity is actually adaptive or not, and to further explain the difference between phenotypic plasticity and adaptive potential. These concepts are mentioned in the proposal, but not explicitly discussed. In addition, it is possible that bsRAD-seq actually includes ddRAD-seq, therefore somewhat confusing the two aims. There also needs to be a clear normalization step between ddRAD and epiRAD. In the description of experimental methods, it would be helpful to fully describe the scale of the populations being studied, and explain how the researchers plan to effectively isolate the species they are targeting. In general, more background was needed for the epigenetics section, and some acronyms used throughout the proposal were never fully defined. The proposal would be stronger if the background information included details about epigenetics and explicitly stated the connection between epigenetic function/diversity and adaptive potential. Not every reviewer will know exactly how epigenetic changes can influence a phenotype.

###### *Broader Impacts*

*Strengths*

The panel thought there were some great ideas in the broader impacts section! Lab visits and internships sound like a great idea to educate the local community, especially young students, about “good microbes.” Public presentations involving hands-on learning and sample collecting will be an effective way to communicate scientific ideas to the public.

*Weaknesses*

In the broader impacts section the researchers could have been more specific about how this research will advance scientific knowledge of the whole ecosystem. One of the strengths of this proposal is that it has strong implications for future climate change research, and it would be nice if that were elaborated on more.

###### *Summary*
Overall the panelists felt this was a strong proposal, but could benefit from more background information.  By expanding the background, especially on the epigenetics portion of the project, the panelists felt that the research design would also be much clearer. The panelists also recommend that this project could be expanded to include sequencing of the diatoms, which could be accomplished with the samples already being collected. All in all, panelists felt that the researcher is asking interesting questions, pursuing novel research, and has strong methods with which to carry it out.
******


### Reviewer 1
Rating: Very good

*What is the intellectual merit of the proposed activity?*

The proposed research has two main aims that will further advance understanding of plankton populations in the Southern Ocean. The first is to measure population structure of planktonic diatoms. The second is to examine genome-wide patterns in epigenetic variation across these locally adapted plankton populations. This study will use plankton samples that were recently collected on a research cruise.
For the first aim, they will use reduced-representation sequencing (specifically ddRAD-seq) to measure population differentiation within two plankton species found in the Southern Ocean. Despite the absence of a reference genome and difficult DNA extraction methods, the PI proposes a feasible solution to the problem, that will allow them to restrict downstream SNP calling in all samples, archival DNA or yet-to-be extracted, to loci genotyped in knock-down axenic samples. Once this is achieved, population structure analyses will be conducted on the neutral data set. This study will be the first to apply modern population genomics methods to eukaryotic phytoplankton.
For the second aim, the research team will develop a diatom-specific reduced representation bisulfite sequencing (RRBS) protocol to investigate genome-wide epigenetic variability across distinct regional diatom populations. The RRBS protocol is advantageous as it does not require a reference genome. This protocol can be adopted by other research labs for similar studies. The proposed research will link specific molecular level processes to spatial patterns in plasticity and local adaptation.

Overall, this study will advance research within eukaryotic plankton ecology. It will lead to more accurate modeling of response to climate change. Development of a new sequencing protocol has the potential to be used by future diatom studies.

The intellectual merit of the study is very strong, but could be enhanced with an explanation of the importance of diatoms. Explaining why diatoms are ecologically important and why we need to study their adaptive potential will strengthen this proposal even more.

*What are the broader impacts of the proposed activity?*

Broader impacts of the proposed activity: Lab members involved in this research will attend local schools during lab hours, host summer internships, and plan annual gatherings/presentations to increase awareness and understanding of the importance of phytoplankton. All these events will engage the public through hands-on activities, including local specimen collection and identification.
These are all great ideas, but they may be difficult to implement. Attending student lab hours for outreach purposes will likely be hard to achieve as schools follow strict lesson plan schedules. Hosting after-school sessions may be easier to schedule with local schools. Additionally, more information about the annual gatherings and presentations is needed. Where will these annual gatherings take place? How will you promote these gatherings? Including more details shows that you have put thought into planning these events and are willing to put them into action.

*Summary Statement*

In summary, I rate this proposal as very good. It has the potential to advance understanding within the field of phytoplankton ecology. I found the development of a new sequencing protocol to be very exciting, as it can potentially be used in future diatom studies. While this is already a strong proposal, it can be further improved with an in-depth explanation of the importance of diatoms. Additionally, more details on the proposed outreach events are necessary to enhance the broader impacts of this study.

Additional notes:
-	The acronyms for a few terms were used without specifying what they stand for. Be sure to clarify every acronym to avoid confusion.



### Reviewer  2
Rating: excellent

*What is the intellectual merit of the proposed activity?*

This proposal seeks to pioneer on ecological and population genomics aspects of eukaryotic phytoplankton in the Southern Ocean within a climate change model framework. Evaluating the future potential adaptation of already cold-adapted polar diatoms to warming seas is vital to advance the understanding of global biochemical cycles, marine productivity and phytoplankton distribution. By examining the abundance and spatial distribution of these photoautotrophs through reduced-representation sequencing and the discovery of epigenetic markers, the authors aim to dissect intra and inter-population mechanisms that can account for the persistence of these microorganisms in the Southern Ocean and their future response to increasing global temperature. According to the authors, this will be the first study to attempt a combination of genome-wide and epigenetic methods to characterize populations of two species of diatoms that differ in their biogeographic distributions as well as in their phylogenetical distances. Building up on previous findings from photosynthetic and asexual organisms, the authors hypothesize that methylated DNA in transposable elements can be a mechanism shaping genetic structure and local or regional adaptation in diatoms.

*What are the broader impacts of the proposed activity?*

Partnering with local environmental organizations and schools in order to increase awareness and understanding of the role microorganisms, such as microscopic algae, play in all ecosystems. Planned activities include day events and short internships for students to get hands-on experience in collecting, isolating and observing the microbial world through state of the art technology like high-powered compound microscopes and SEMs. The authors will also organize annual events to reach out to the non-specialist community with the overall goal of exposing them to the fascinating and rarely seen world of the microbial photoautotroph.

*Summary Statement*

This is an excellent proposal with huge potential to provide major insights into the genomics mechanisms that underlie phytoplankton´s polymorphisms and  potential adaptation to global warming. I would like to pose some general questions to the authors for the sole purpose of pinpointing some potential issues with the methodological approaches. Are populations of these two species maintained mainly through sexual or asexual reproduction? In the case asexual reproduction is predominant, how do the authors think clonal populations could affect their exploration of  genetic structure and the distinction between neutral and outlier loci that rely on estimating allele frequencies within and between populations ? Do authors think that they could encounter any issue in the identification of transposable and repeated elements when working with relatively short reads, as it is normally the case with reduced-representation sequencing? Lastly, given the high dispersal typically exhibited by planktonic organisms, authors should consider in a more detailed manner how  currents and thermohaline circulation patterns in the area or region could influence gene flow in the species under study.

### Reviewer 3

*What is the intellectual merit of the proposed study?*

The proposed study aims to apply NGS reduced representation techniques to determine population structure and epigenetic structure or variation in Southern Ocean planktonic diatoms.

There is a clear knowledge gap that this proposal wants to address, the lack of adoption of NGS techniques in the field of planktonic diatoms is obvious and needs to be remedied. The goal to lay the framework for further research involving genomic techniques in diatom research is ambitious, but necessary. Fields that do not jump into the area of genomics are at risk of being overshadowed, and as sequencing costs continue to decrease, there is no reason to not adopt those methods. Understanding of the extent of population connectivity over such a large scale would lead to understanding of not only the diatoms under study, buy also broader oceanographic processes and potentially the connectivity of organisms that have life history stages that are similar to planktonic diatoms (ex. planktonic larvae). There is the potential for expanding the results from this study into other organisms.

As a companion to the hypothesized population structure, there may also be epigenomic structure or differentiation between the various sampled locations. If there is less signal within the genetic structure than proposed hypothesis, variation between groups may still be explained by epigenetic differences. It will be a challenge to separate out between-species epigenetic variation that may cloud the location-specific signals. However, that is completely unknown at this time if different species of diatoms have different epigenetic patterns based on species not environment, so even if that is a result of this study, it is an important discovery. The proposed methods are also well thought out to get around the problems with working with organisms that do not have very many, if any, genomic resources. While they may not end up giving the resolution for the questions proposed, ie. whether the restriction enzymes are expected to capture transposable elements or not, this study will still provide a base for further investigation into those topics if needed.

The problem of separating out species in population connectivity or structure analysis is not well addressed, it is unclear if clusters are expected by species or across species. However, these assumptions may not be known, and the authors do propose to create a standardized process for investigating population structure in diatoms. There just seems to be a serious question about the scale of how populations are going to be considered that needs to be stated clearer.

*What are the broader impacts of this study?*

While the goals to engage the community in understanding of good diatoms and microbes is very important, the current media these days is flooded by questionable science on microbiomes and heavily-focused negative media on harmful microbial blooms, the plans for outreach need to be thought out further. It is a supreme challenge to get the public engaged in organisms that are microscopic, I wonder if “exclusive” lab visits or viewing diatoms through impressive, but intimidating, high-powered microscopes are the best avenue for teaching about organisms that are already an unknown and seem separate from most peoples’ lives. This study would benefit from further thinking about how to present diatoms in a more accessible way.

*Summary Statement*

The proposed study has a really solid foundation for investigating a serious knowledge gap apparent in the lack of any genomic recourses or techniques that have been applied to SO diatoms. The added aim to investigate differences in the epigenome of diatoms that may contribute to differential adaptation or plasticity of organisms in a changing environment is just as important as the first aim to determine population structure. Some things to consider would be how to determine groups in the diatoms, I would imagine that is it impossible to separate out species before DNA extraction. The mix of species may cause a large amount of noise in the sequence data that could cloud any genomic or epigenomic signals based on location/region. However, this is a challenge of the field that would be faced by any research focused on diatoms. If the researchers could come up with a procedure for separating out those factors within the genetic data, that would open up the use of genomic data on diatoms incredibly. The authors would be contribution immensely in terms of methods as well as results of the study.
